userguide/tutorials/metrics-datadog.adoc

```diff
@@ -0,0 +1,163 @@
+= Managing Metrics with Datadog
+
+:imagesdir: https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws
+
+
+== Single Tier and Multitier Implementations
+
+Measurements and metrics are an important part of managing your Kill
+Bill system to be sure that performance is optimized and any problems
+can be detected early. Kill Bill collects over 1300 metrics to show you
+every conceivable view of your system's behavior.
+
+Several third party tools are available to manage and display metrics of
+this type. One popular tool is https://www.datadoghq.com/[datadog]. This
+package offers a low-cost option to monitor and display whatever metrics
+you may find useful from the Kill Bill collection, including metrics
+that show the performance and resource use of Kill Bill and Kaui
+themselves, and metrics describing the infrastructure on which these
+systems run.
+
+Datadog can be integrated with several types of Kill Bill installations. In this article we discuss the use of datadog to manage metrics produced
+by two types of implementations on Amazon Web Services (AWS):
+Single Tier and Multitier. With the latest Amazon Machine Image (AMI)
```

Comment:
@jdmooney @pierre - I see both occurrences in the docs.killbill.io, but perhaps we're in the middle of switching? To me, it's difficult to read "multitier" because of the repetition of the "ti."

Reply:
I'll defer to what you suggest. But we'll have to change a number of docs for consistency.

Reply:
ok thanks.